src/Command/InstallCommand.php

```diff
@@ -47,6 +47,8 @@ class InstallCommand extends AbstractLockedCommand
      * @var array
      */
     private $emptyDirs = [
+        'app',
+        'app/config',
```

Comment:
Should be fixed by using `Filesystem` in https://github.com/contao/installation-bundle/pull/20/commits/0c871edc068426d5f807a417308bb5c7a7ee1597#diff-7d35f688c872d8ff73ee0dd8ff9e353eR97 as it will make sure the target folder exists.